core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -119,7 +119,7 @@ private RpcUtils() {} // prevent instantiation
         MetricsSystem.counter(getQualifiedFailureMetricName(methodName)).inc();
         if (!logger.isDebugEnabled()) {
           logger.warn("Exit (Error): {}: {}, Error={}", methodName,
-              String.format(description, args), e);
+              String.format(description, args), e.getMessage());
```

Comment:
@gpang Looks like they change from printing only the message to print all the stack trace in logger.warn.
Can you help confirm that @HelloHorizon? 
Maybe we need to change other warn loggings in the whole system as well.

Reply:
@LuQQiu They print the stack trace in 2.x as follows:
```
warn(CharSequence message, Throwable t)
         Logs a CharSequence at the WARN level including the stack trace of the Throwable t passed as parameter.`
And for 1.x as follows:
```
warn(Object message, Throwable t)
          Log a message with the WARN level including the stack trace of the Throwable t passed as parameter.
```
Both of them throw the stack trace?

Reply:
In the logs themselves, do the stacks shows up for both versions?

Reply:
The stacks are not showing up in the previous version but for the latest version.

Reply:
@gpang - based on the comment from @HelloHorizon , this should be good to go - please confirm.